src/embed/connection.rs

```diff
@@ -189,9 +189,9 @@ async fn execute<'a, 'b>(
         "closeTransaction" => return to_js(do_close(ctx, from_js(data)?).await),
         "setLogLevel" => return to_js(do_set_log_level(ctx, from_js(data)?).await),
 
-        "push" => return to_js(do_push(ctx, from_js(data)?).await),
-        "beginPull" => return to_js(do_begin_pull(ctx, from_js(data)?).await),
-        "maybeEndPull" => return to_js(do_maybe_end_pull(ctx, from_js(data)?).await),
+        "push" => return to_js(do_try_push(ctx, from_js(data)?).await),
```

Comment:
Not clear if we also want "try" in the RPC? I think yes for consistency.

Reply:
yes agree